mantidproject/mantid
# New POWGEN IDF

Description of work.

**To test:**

Load it in and see how the instrument works. It is already imbedded in `PG3_37616` which is a diamond run.

*There is no associated issue.*

Merge [mantidgeometry#93](https://github.com/mantidproject/mantidgeometry/pull/93) after this is accepted.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
